gradient/cli/experiments.py

```diff
@@ -861,9 +861,9 @@ def experiment_remove_tags(id, options_file, api_key, **kwargs):
     "--metric",
     "metrics_list",
     multiple=True,
-    type=ChoiceType(constants.METRICS_MAP, case_sensitive=False),
+    type=str,
     default=(constants.BuiltinMetrics.cpu_percentage, constants.BuiltinMetrics.memory_usage),
-    help="One or more metrics that you want to read. Defaults to cpuPercentage and memoryUsage",
+    help=("One or more metrics that you want to read. Defaults to cpuPercentage and memoryUsage. Custom metrics are {}".format("hello")), # TODO replace "hello" with custom metrics
```

Comment:
Let's instruct the user to use `gradient metrics experiment metrics list` to see available custom metrics, and instead we should show the user the contents of `constants.METRICS_MAP`.

Reply:
ohh nice then i'm 5 min away from completing this